roman_numeral.js

```diff
@@ -0,0 +1,55 @@
+/* Convert the given number into a roman numeral.*/
+
+/**
+ * @param {int} num
+ * @return {array}
+ * depending on the spec, returns an array with num details
```

Comment:
The spec is not a dependency for the function it's hard-coded in it. What does details mean? I think it's a decomposition in the Roman numerals base that you are returning.